### ISSUE TITLE
linux-boundary: parse error

### ISSUE THREAD

**Issue by texierp** [Freescale/meta-freescale-3rdparty#45]
@chrisdimich since your commit 0ad69f4bd93637606c5fd7fdac38505916a5ba42 (linux-boundary: Move to version 4.14)

I have the following issue if I try to build an image for the `nitrogen8m`:
```
ERROR: /home/bdx-iot/oe/build/../sources/meta-freescale-3rdparty/recipes-kernel/linux/linux-boundary_4.14.x.bb: Error during parse shell code, the last 5 lines are:                    | ETA:  --:--:--
		else
			bberror "KERNEL_DEFCONFIG does not exist!" Please set a valid KERNEL_DEFCONFIG"
		fi
	fi
```

The fix is trivial:

```
-           bberror "KERNEL_DEFCONFIG does not exist!" Please set a valid KERNEL_DEFCONFIG"
+           bberror "KERNEL_DEFCONFIG does not exist! Please set a valid KERNEL_DEFCONFIG"
```

But how was this tested?

**Comment by chrisdimich** [comment]
Hi Pierre-jean,

Yes, we meant to push the working version. It was a mistake. We are fixing
it now. Please stay tuned.

Thanks,

Chris Dimich
Boundary Devices, LLC
www.boundarydevices.com
7 Orchard Road, Suite 102*｜*Lake Forest, CA 92630

**Comment by texierp** [comment]
>  Yes, we meant to push the working version. It was a mistake. We are fixing it now. Please stay tuned.

Thanks @chrisdimich ;)

**Comment by chrisdimich** [comment]
HI Pierre-jean,

Okay I have put it as part of this pull request:

https://github.com/Freescale/meta-freescale-3rdparty/pull/46

Thanks,

Chris Dimich
Boundary Devices, LLC
www.boundarydevices.com
7 Orchard Road, Suite 102*｜*Lake Forest, CA 92630

**Comment by texierp** [comment]
Fixed by afc1c4b77dfa8feaa4e69cfa280f10d0ac7f6d67
